### PR TITLE
Draft intro for multiple models 

### PR DESCRIPTION
**Pull request by JoelPasvolsky**

### CODE REVIEW

File: docs/intro/intro_cqm.rst [hunk 1]
```diff
@@ -13,7 +13,47 @@ typically formulate the constraints as penalties. Constrained models such as
...
+This example constructs a CQM from symbolic math, which is especially useful for
+learning and testing with small CQMs.
+
+>>> x = BINARY('x')
+>>> y = INTEGER('y')
+>>> cqm = dimod.CQM()
+>>> objective = cqm.set_objective(x+y)
+>>> cqm.add_constraint(y <= 3)
+
+For very large BQMs, you might read the data from a file or construct from a NumPy
```
Comment (arcondello) [inline docs/intro/intro_cqm.rst:58]: Do you mean BQM or CQM here? I would probably use the generic "model".

**Review by arcondello — APPROVED** [review]

File: docs/intro/intro_cqm.rst (hunk 1 above)
Comment (JoelPasvolsky) [reply docs/intro/intro_cqm.rst]: Yes, copy & paste error. Fixed

File: docs/intro/intro_cqm.rst [hunk 2]
```diff
@@ -13,7 +13,47 @@ typically formulate the constraints as penalties. Constrained models such as
...
 objective and its set of constraints, as models or in symbolic form.
 
-CQM
-===
+Supported Models
+================
 
-TODO
+dimod provides a :term:`constrained quadratic model` (CQM) class that encodes
+a quadratic objective and one or more quadratic equality and inequality constraints.
```
Comment (arcondello) [inline docs/intro/intro_cqm.rst:20]: technically there can be no constraints.

File: docs/intro/intro_quadratic.rst [hunk 3]
```diff
@@ -15,31 +15,63 @@ When using such samplers to handle problems with constraints, you typically
...
+  For the QM class, its attributes and methods, see :ref:`qm`.
 
-For an introduction to BQMs, see :std:doc:`Binary Quadratic Models <oceandocs:concepts/bqm>`.
+* **Binary Quadratic Models**
 
-dimod can represent BQMs with classes suited to different needs for
-performance and ease of use. Beginners might start with the simple Python
-:ref:`AdjDictBQM <adjdictbqm_dimod>` class (:class:`dimod.BinaryQuadraticModel`),
-and later switch to a higher-performing class with a C++ implementation.
+  The :term:`binary quadratic model` (BQM) class, :class:`dimod.BinaryQuadraticModel`,
```
Comment (arcondello) [inline docs/intro/intro_quadratic.rst:33]: why not make the term a link to the glossary? Or does that happen automatically

**Review by arcondello — APPROVED** [review]

File: docs/intro/intro_quadratic.rst (hunk 3 above)
Comment (JoelPasvolsky) [reply docs/intro/intro_quadratic.rst:33]: The `:term:` creates a link to the glossary. You can see the mutual links in the glossary terms themselves here: https://docs.ocean.dwavesys.com/en/stable/concepts/index.html
An `binary quadratic model` is the first glossary entry

File: docs/intro/intro_cqm.rst (hunk 2 above)
Comment (JoelPasvolsky) [reply docs/intro/intro_cqm.rst]: Added "possibly"

**Pull request merged**